src/subcommands/list_keys.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Lists all keys belonging to the application.
+
+pub use crate::cli::ParsecToolApp;
+use crate::error::ParsecToolError;
+use crate::subcommands::ParsecToolSubcommand;
+use parsec_client::core::interface::requests::ProviderID;
+use parsec_client::core::operation_client::OperationClient;
+use parsec_interface::operations::list_keys;
+use parsec_interface::operations::{NativeOperation, NativeResult};
```

Comment:
Oh yes I missed that, please use the interface from the client at `parsec_client::core::interface` like above.